GameWorld/HexGrid/DrawableHexSphere.cs

```diff
@@ -114,6 +122,46 @@ public void SetGroundHeight(int groundHeight)
             InitializeVertices();
         }
 
+        public double GetHeightFromCoveragePercent(int coveragePercent)
+        {
+            double resultHeight = 0.5f;
+            List<double> tiles_heights = new List<double>();
+
+            foreach (var t in _sphereGrid.Tiles)
+            {
+                tiles_heights.Add(t.Height);
+            }
+
+            tiles_heights.Sort();
+
+            if (0 > coveragePercent || 100 < coveragePercent)
```

Comment:
Please, don't use "constant-first" notation in comparison. Use coveragePercent < 0 || coveragePercent > 100